zcash_client_backend/src/data_api.rs

```diff
@@ -437,11 +439,17 @@ impl BlockMetadata {
         self.block_hash
     }
 
-    /// Returns the size of the Sapling note commitment tree as of the block that this
+    /// Returns the size of the Sapling note commitment tree as of the end of the block that this
     /// [`BlockMetadata`] describes.
```

Comment:
```suggestion
    /// Returns the size of the Sapling note commitment tree for the final treestate of the block
    /// that this [`BlockMetadata`] describes.
```
(This is the [terminology used in the protocol spec](https://zips.z.cash/protocol/protocol.pdf#transactions).)